ocp_resources/configmap.py

```diff
@@ -1,34 +1,23 @@
-from ocp_resources.constants import TIMEOUT_4MINUTES
 from ocp_resources.resource import NamespacedResource
 
 
 class ConfigMap(NamespacedResource):
     """
-    Configmap object
+    https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.26/#configmap-v1-core
     """
 
     api_version = NamespacedResource.ApiVersion.V1
 
     def __init__(
         self,
-        name=None,
-        namespace=None,
         data=None,
-        teardown=True,
-        client=None,
-        yaml_file=None,
-        delete_timeout=TIMEOUT_4MINUTES,
         **kwargs,
     ):
-        super().__init__(
-            name=name,
-            namespace=namespace,
-            client=client,
-            teardown=teardown,
-            yaml_file=yaml_file,
-            delete_timeout=delete_timeout,
-            **kwargs,
-        )
+        """
+        Args:
+            data (dict): key-value configuration pairs.
```

Comment:
should raise if missing (this is mandatory right?)

Reply:
Done